### DISCUSSION TITLE
Get slow when formatting with eslint

### DISCUSSION THREAD

**Discussion by chojs23** [LazyVim/LazyVim#1957] [Q&A] [answered]
I enabled two extra lazyvim plugins
`    { import = "lazyvim.plugins.extras.linting.eslint" },
    { import = "lazyvim.plugins.extras.formatting.prettier" },
`

It's fine with small project but get laggy when editing large typescript projects. I enabled format on save, so every time I save my neovim stuck about 0.5 seconds ~ 1 seconds.

This only happens with typescript projects, my rust and php projects are fine with formatting.

**Comment by chojs23** [comment, marked as answer]
FYI : the prettier format takes long time not eslint. I changed to use prettierd and it formats instantly.

**Reply by amritk** [reply]
hey whats the config to change to prettierd? I'm new to lazyvim (coming from chad)
